### ISSUE TITLE
Show site URI on "View Item" screen

### ISSUE THREAD

**Issue by GeoDosch** [bitwarden/browser#508]
On the View Item screen, there is a 'Website' field, but it only shows the domain of the site.  If I want to see what URI BW has saved, I need to go to the Edit Item screen.

When a new site is created, the URI that gets stored is often specific to account creation, email validation, etc.  I like to clean them up, so that I don't get error messages or other strange pages when I try to launch the site from BW.  It would be nice if I could quickly check the saved URI for a given site without the need to go into the Edit screen.

**Comment by kspearrin** [comment]
URIs can often be hundred if not thousands characters, which is why we chose to limit it to the hostname of the site here. Displaying the full URI would create UI issues.

**Comment by GeoDosch** [comment]
I'm not sure if there's an internal difference with how it displays on the View Item vs. Edit Item screen.  The latter doesn't appear to have issues with long URIs.  If very long strings are a problem on the View Item screen, then limiting it to showing the first 'x' number of characters, in order to prevent UI issues, would still be helpful, as that's usually all that's needed to see if it's a GET string for sign-up, etc.

**Comment by kspearrin** [comment]
Yes, it is different. On the edit page it is an input textbox, which overflows with input based on your cursor.

**Comment by kriswilk** [comment]
I would also prefer if the view panel would show the actual URI -- or at least truncated to X characters followed by ". . ." so that I know there is more to it than just the domain. Right now the "name" and "website" usually end up looking identical even though the "website" is a more complex URI.

Couldn't that field be changed to a read-only edit box but styled to blend in with the rest of the UI?

@GeoDosch, if it helps any, mousing over the URI in the view panel should get you a tooltip with the full URI. At least in Chrome.

**Comment by GeoDosch** [comment]
> if it helps any, mousing over the URI in the view panel should get you a tooltip with the full URI. At least in Chrome
@kriswilk, yes, that helps tremendously!  I don't know how I missed that.  Thank you!
